go.mod

```diff
@@ -10,6 +10,7 @@ require (
 	github.com/google/uuid v1.2.0
 	github.com/kr/pretty v0.1.0
 	github.com/kr/text v0.1.0 // indirect
+	github.com/stretchr/testify v1.7.0
```

Comment:
Let's not bring in the dep, govmomi tests just use the stdlib testing package.

Reply:
Thanks for the review, @dougm. I have removed the dependency and updated the test to use stdlib testing package.